File: 브론즈 3 (3문제)/2857_FBI.c
#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>
#include <string.h>  // strstr

int main() {
	int count = 0;
	for (int i = 1; i <= 5; i++) {
		char name[11];

		// 문자열 입력받기
		scanf("%s", name);

		// 문자열에 FBI가 포함되면 해당 요원 번호 출력
		if (strstr(name, "FBI")) {
			printf("%d\n", i);
		}
		else {
			count++;
		}
	}

	// FBI가 존재하지 않는 경우
	if (count == 5) {
		printf("HE GOT AWAY!");
	}

	return 0;
}

File: 브론즈 3 (3문제)/2857_FBI.py
count = 0
for i in range(1, 6):
    # 이름 입력받기
    name = input()

    # FBI 문자열이 발견되는 경우 요원 번호 출력
    if name.find("FBI") != -1:
        print(i)
    # 발견되지 않는 경우 count 변수 1 증가
    else:
        count += 1

# FBI 문자열이 하나도 발견되지 않는 경우 정해진 문자열 출력
if count == 5:
    print("HE GOT AWAY!")
